packages/styles/dropdown.css

```diff
@@ -1,44 +1,3 @@
-.Dropdown {
-  position: absolute;
-  padding: 8px;
-  background-color: var(--white);
-  box-shadow: 0px 0px 2px 1px rgba(0, 0, 0, 0.3);
-  border: 1px solid var(--field-border-color);
-  color: var(--text-color-base);
-  display: none;
-  list-style: none;
-}
-
-.Dropdown.Dropdown--active {
-  display: block;
-}
```

Comment:
Should this be removed, too? https://github.com/dequelabs/cauldron/blob/7fd7d095bc8c85c5369371baaf74c7300c578fd2/packages/react/src/components/TopBar/TopBarMenu.tsx#L104

Reply:
No, this prevents a breaking change for someone using that class.